en/subscription_services_privacy_notice.md

```diff
@@ -1,55 +1,67 @@
+ 
 
 ### Firefox Relay {: #relay }
 
-__Interaction data.__ Mozilla receives data about your interactions with the Services. For instance, when you log in and out and the preferences you set; learn more about [Relay interaction data](https://github.com/mozilla/fx-private-relay/blob/main/METRICS.md). We use Interaction data to improve performance and stability for our users, and to measure the Services’ performance.
+__Interaction data.__ Mozilla receives data about your interactions with Firefox Relay, for instance, when you log in and out and the preferences you set. Learn more about [Relay interaction data](https://github.com/mozilla/fx-private-relay/blob/main/METRICS.md); you can opt-out of this collection by turning on the [Do Not Track (DNT)](https://support.mozilla.org/kb/how-do-i-turn-do-not-track-feature) feature in your browser. 
 
-In addition, to help understand the effectiveness of our marketing campaigns, Mozilla may receive your IP address in addition to campaign data and referral data like what ad campaigns you engaged with and your operating system, device type, device identifiers, and operating system version. Mozilla shares this with Adjust, but we do not share or store your IP address. Read the [Adjust documentation](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/adjust/adjust.md).
+More generally, Mozilla may also use cookies, device information, and IP addresses, along with cookies and third party services to help us understand in the aggregate how users engage with our products, services, communications, websites, online campaigns, snippets, devices, and other platforms. We use:
 
-__Technical data.__ Mozilla receives basic information about the installed version of your Relay Add-on software, and the device they are installed on, including the operating system and hardware configuration. Across Subscription Services, your IP address is temporarily collected as part of our server logs for 90 days. We use technical data to improve performance and stability for our users, and to measure the Services’ performance.
+* Google Analytics, which places a cookie on your device, to obtain metrics on how users engage with our websites. This helps us to improve site content. To prevent Google Analytics from collecting data about your use of Mozilla Monitor, you can install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout).
+
+__Technical data.__ Mozilla receives basic information about the installed version of your Relay Add-on software, and the device they are installed on, including the operating system and hardware configuration. Across Subscription Services, your IP address is temporarily collected as part of our server logs for 90 days. We use Technical data to improve performance and stability for our users, and to measure the Services’ performance.
```

Comment:
- Lower case `t` in `Technical data`
- Lower case `s` in `Services’ performance`